src/Datadog.Trace.ClrProfiler.Native/cor_profiler.cpp

```diff
@@ -2840,15 +2841,17 @@ size_t CorProfiler::CallTarget_RequestRejitForModule(ModuleID module_id, ModuleM
            ", Method=", caller.name, 
            ", Signature=", caller.signature.str(),
            "]");
+
+      rejit_handler->EnqueueForRejit(module_id, methodDef);
       
       enumIterator = ++enumIterator;
     }
   }
 
   // Request the ReJIT for all integrations found in the module.
   if (!vtMethodDefs.empty()) {
```

Comment:
Don't forget to remove vtMethodDefs and vtModules in the final version ;)

Reply:
yeah, I'm working on that, also to accept the complete array so we don't suspend the runtime for each single method.

Reply:
the method was changed in: a029fef